<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddFieldDialog</class>
 <widget class="QDialog" name="AddFieldDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>200</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add field</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="layout_1">
    <property name="margin">
     <number>10</number>
    </property>
    <item>
     <layout class="QFormLayout" name="formLayout_1">
      <item row="0" column="0">
       <widget class="QLabel" name="labelName">
        <property name="text">
         <string>Field name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="name"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelType">
        <property name="text">
         <string>Type </string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QVBoxLayout" name="layout_2">
        <item>
         <widget class="QRadioButton" name="radioHandwritten">
          <property name="text">
           <string>Handwritten</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioPrinted">
          <property name="text">
           <string>Printed</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioMarked">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Mark</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
      <property name="centerButtons">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
